runelite-client/src/main/java/net/runelite/client/plugins/raids/RaidsPointsOverlay.java

```diff
@@ -64,22 +65,43 @@ public Dimension render(Graphics2D graphics)
 		int totalPoints = client.getVar(Varbits.TOTAL_POINTS);
 		int personalPoints = client.getVar(Varbits.PERSONAL_POINTS);
 
+
 		panel.getChildren().clear();
 		panel.getChildren().add(LineComponent.builder()
-			.left("Total:")
-			.right(POINTS_FORMAT.format(totalPoints))
-			.build());
+				.left("Total:")
+				.right(POINTS_FORMAT.format(totalPoints))
+				.build());
 
 		panel.getChildren().add(LineComponent.builder()
-			.left(client.getLocalPlayer().getName() + ":")
-			.right(POINTS_FORMAT.format(personalPoints))
-			.build());
+				.left(client.getLocalPlayer().getName() + ":")
+				.right(POINTS_FORMAT.format(personalPoints))
+				.build());
 
 		panel.getChildren().add(LineComponent.builder()
-			.left("Party size:")
-			.right(String.valueOf(client.getVar(Varbits.RAID_PARTY_SIZE)))
-			.build());
+				.left("Party size:")
+				.right(String.valueOf(client.getVar(Varbits.RAID_PARTY_SIZE)))
+				.build());
+
+		if (totalPoints == 0) // Calculates average combat level of the party before the raid has begun
+		{
+			panel.getChildren().add(LineComponent.builder()
+					.left("Average combat:")
+					.right(String.valueOf(averageCombat()))
+					.build());
+
+		}
 
 		return panel.render(graphics);
 	}
-}
+
+	private int averageCombat()
+	{
+		int sumOfCombatLevels = 0;
+
+		for (Player currPlayer :client.getPlayers())
```

Comment:
Add a space after : please.

Also I am wondering how client.getPlayers() works because wouldn't this add all the players visible and not just in your raid group? This is only accurate once inside the raid. Wouldn't it be more accurate if you totaled the combats of all players in the CC?

Reply:
Yup, only people inside the raid, but on the other hand you sometimes have people not wishing to join that are in the cc. Take a large clan for example.

Reply:
Fair enough. I guess this is good enough for checking after the group is made, I just figured it would be better to know before you get into the raid itself. Also if people are off screen it will bug.